gucci.go

```diff
@@ -47,10 +47,13 @@ func main() {
 		tplPath := c.Args().First()
 		vars, err := loadVariables(c)
 		if err != nil {
-			return err
+			return cli.NewExitError(err, 14)
```

Comment:
What the reason for exit code 14 here?

Same questions for 32 I guess

Reply:
Yeah, I picked random numbers. I switched them to 1 as if we're going to do different error codes it'd be nice to not have there be just 2 options. (e.g. return a cli.NewExitError further down in the code.)